spec/dummy/db/migrate/20220603155503_add_save_and_exit_path_to_onboarding_session.rb

```diff
@@ -0,0 +1,5 @@
+class AddSaveAndExitPathToOnboardingSession < ActiveRecord::Migration[5.2]
+  def change
+    add_column :onboarding_sessions, :save_and_exit_path, :string
```

Comment:
Where are we using this?

Reply:
@kcpavan This was used when we had some issue in the save & exit path. Since we are using the super method to save and exit we no longer need this now.